### PR TITLE
[5.0] Fix name of parameter when changing group

### PR DESCRIPTION
**Pull request by chrisburr**
```python
Internal Server ErrorTraceback (most recent call last):

  File "/opt/dirac/versions/v11.0.0a2-1653977607/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py", line 649, in executeMethod
    return self.methodObj(*args, **kwargs)

TypeError: web_changeGroup() missing 1 required positional argument: 'group'


During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "/opt/dirac/versions/v11.0.0a2-1653977607/Linux-x86_64/lib/python3.9/site-packages/tornado/web.py", line 1704, in _execute
    result = await result

  File "/opt/dirac/versions/v11.0.0a2-1653977607/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py", line 968, in get
    await self.__execute(*args, **kwargs)

  File "/opt/dirac/versions/v11.0.0a2-1653977607/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py", line 921, in __execute
    self.__result = await IOLoop.current().run_in_executor(None, partial(self.executeMethod, args, kwargs))

  File "/opt/dirac/versions/v11.0.0a2-1653977607/Linux-x86_64/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)

  File "/opt/dirac/versions/v11.0.0a2-1653977607/Linux-x86_64/lib/python3.9/site-packages/WebAppDIRAC/Lib/WebHandler.py", line 225, in executeMethod
    return super().executeMethod(args,kwargs)

  File "/opt/dirac/versions/v11.0.0a2-1653977607/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py", line 654, in executeMethod
    raise HTTPError(HTTPStatus.INTERNAL_SERVER_ERROR, str(e))

tornado.web.HTTPError: HTTP 500: Internal Server Error (web_changeGroup() missing 1 required positional argument: 'group')
```

BEGINRELEASENOTES

FIX: Changing group or setup

ENDRELEASENOTES

### CODE REVIEW

**Comment by TaykYoku** [comment]
this fix also here #656

File: src/WebAppDIRAC/WebApp/handler/RootHandler.py [hunk 1]
```diff
@@ -151,10 +151,10 @@ def web_loginComplete(self, code, state, **kwargs):
                 t.generate(next=authSession["next"], access_token="", message=result["Message"]).decode()
             )
         nextURL = f"/{Conf.rootURL().strip('/')}"
-        if group := result["Value"].get("group"):
-            nextURL += f"/g:{group}"
         if setup := self.getUserSetup():
             nextURL += f"/s:{setup}"
+        if group := result["Value"].get("group"):
+            nextURL += f"/g:{group}"
```
Comment (chrisburr) [inline src/WebAppDIRAC/WebApp/handler/RootHandler.py:157]: @TaykYoku Why did you change this ordering in https://github.com/DIRACGrid/WebAppDIRAC/pull/654? The routing only understands if the setup comes first:

https://github.com/DIRACGrid/WebAppDIRAC/blob/d70c268ff47fc051ca29fd0415fcaeb0e7587f2a/src/WebAppDIRAC/Core/HandlerMgr.py#L37-L40

Comment (TaykYoku) [reply src/WebAppDIRAC/WebApp/handler/RootHandler.py:157]: Yes I know. I have already submitted a PR with all the corrections. https://github.com/DIRACGrid/WebAppDIRAC/pull/656

Comment (chrisburr) [reply src/WebAppDIRAC/WebApp/handler/RootHandler.py:157]: Except that PR was for a different issue and made no mention of what "bugs" it was fixing. How was I supposed to know the solution the group changing issue was hidden in that PR so my debugging was a complete waste of time?

Comment (chrisburr) [reply src/WebAppDIRAC/WebApp/handler/RootHandler.py:157]: Note: It still makes no mention of what the PR does beyond the delightful "fix bugs".

Comment (TaykYoku) [reply src/WebAppDIRAC/WebApp/handler/RootHandler.py:157]: Yeap sorry, I forgot to add a list of fixes to the release notes description, now fixed.

**Review by TaykYoku — APPROVED** [review]
